blog/modules/ROOT/static/GC-bench.sc

```diff
@@ -0,0 +1,25 @@
+val liveSets = Seq(400, 800, 1600, 3200, 6400)
+val heapSizes = Seq(800, 1600, 3200, 6400, 12800)
+val lists = for (liveSet <- liveSets) yield {
+  for (heapSize <- heapSizes) yield {
+    if (liveSet >= heapSize) ("", "")
+    else {
+      println(s"Benchmarking liveSet=$liveSet heapSize=$heapSize")
+      val javaBin = os.Path(sys.env("JAVA_HOME")) / "bin"/"java"
+      val res = os.proc(javaBin, s"-Xmx${heapSize}m", "-XX:+UseZGC", "GC.java", liveSet, 10000, 5)
```

Comment:
```suggestion
      val res = os.proc(javaBin, s"-Xmx${heapSize}m", "-XX:+UseZGC", "-XX:+ZGenerational", "GC.java", liveSet, 10000, 5)
```
Isn't it better to see how the Generational mode of ZGC behaves?

Reply:
As of Java 23 Generational mode is the default and non-generational mode is deprecated [JEP 474: ZGC: Generational Mode by Default](https://openjdk.org/jeps/474)

Reply:
Indeed. So my point was we should rather run that, non-generational ZGC is irrelevant at this point.

Is that the case? Are we running the benchmark against OpenJDK >= 23? If yes, than that's great, I apologize for my misunderstanding. I thought it's being run against OpenJDK 21, because the article links to 21 docs https://docs.oracle.com/en/java/javase/21/gctuning/z-garbage-collector.html